tests/integration/monitors/run.rs

```diff
@@ -2,6 +2,8 @@ use crate::integration::{mock_endpoint, register_test, EndpointOptions};
 
 #[test]
 fn command_monitors_run() {
+    // TODO: How do I turn off token auth and force DSN auth in the tests?
```

Comment:
Right now we set all of these variables upfront when registering the test 

https://github.com/getsentry/sentry-cli/blob/a8a076356c82ae432f0e629956d8eecb6b1f0f6c/tests/integration/mod.rs#L32-L40

Unfortunately we can't __unset__ these variables as far as I can tell.

We may need to update `register_test` to have some options such as `auth_type`.

Reply:
Fixed this in https://github.com/getsentry/sentry-cli/pull/1543